integration-cli/docker_cli_save_load_unix_test.go

```diff
@@ -29,9 +29,8 @@ func (s *DockerSuite) TestSaveAndLoadRepoStdout(c *check.C) {
 	saveCmd := exec.Command(dockerBinary, "save", repoName)
 	saveCmd.Stdout = tmpFile
 
-	if _, err = runCommand(saveCmd); err != nil {
-		c.Fatalf("failed to save repo: %v", err)
-	}
+	_, err = runCommand(saveCmd)
```

Comment:
Can we use `dockerCmd` instead? It checks for errors and gives a nicely formatted error message on failure.